cellpack/autopack/ingredient/agent.py

```diff
@@ -25,16 +25,16 @@ def __init__(
         self.partners = partners
         self.packing_mode = packing_mode
 
-        assert self.packing_mode in [
-            "random",
-            "close",
-            "closePartner",
-            "randomPartner",
-            "gradient",
-            "hexatile",
-            "squaretile",
-            "triangletile",
-        ]
+        # assert self.packing_mode in [
+        #     "random",
+        #     "close",
+        #     "closePartner",
+        #     "randomPartner",
+        #     "gradient",
+        #     "hexatile",
+        #     "squaretile",
+        #     "triangletile",
+        # ]
```

Comment:
something weird is going on with packing_mode. I would love some help on this if possible. i added a migrate script to make sure it's defined, and still hit this error

Reply:
I ran the packing and the result looked good! What error are you seeing, or is there a way I can reproduce it?

Reply:
if you uncomment these lines